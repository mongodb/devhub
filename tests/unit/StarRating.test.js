import React from 'react';
import { shallow } from 'enzyme';

import StarRating from '~components/dev-hub/star-rating';

const props = {
    clickHandlers: [jest.fn()],
};

describe('StarRating', () => {
    let shallowWrapper;

    beforeAll(() => {
        shallowWrapper = shallow(<StarRating {...props} />);
    });

    it('renders correctly', () => {
        expect(shallowWrapper).toMatchSnapshot();
    });
});
