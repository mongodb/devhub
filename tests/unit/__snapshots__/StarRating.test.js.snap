// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StarRating renders correctly 1`] = `
<ArticleRatingProvider
  value={
    Object {
      "ratingState": Object {
        "isClicked": false,
        "starRatingFlow": null,
        "starts": Array [
          false,
          false,
          false,
          false,
          false,
        ],
      },
    }
  }
>
  <Memo(StarRating)
    clickHandlers={
      Array [
        [MockFunction],
        [MockFunction],
        [MockFunction],
        [MockFunction],
        [MockFunction],
      ]
    }
  />
</ArticleRatingProvider>
`;
